### ISSUE TITLE
wandb.Run object has no attribute 'close'

### ISSUE THREAD

**Issue by WeihongPan** [zjunlp/DeepKE#83]
example/ner/few_shot/run.py main函数中最后一行 writer.close() 
wandb.Run 对象没有close方法

另外，setup.py依赖项的倒数第二行'wandb==0.12.7'少了个逗号

**Comment by WeihongPan** [comment]
PPS. example/ner/few-shot/conf/train/few_shot.yaml 的bart_name的值应为"facebook/bart-large" 否则报错：

OSError: Model name '/data/lilei/project/BARTNER-AMAX/facebook/bart-large' was not found in tokenizers model name list (facebook/bart-base, facebook/bart-large, facebook/bart-large-mnli, facebook/bart-large-cnn, facebook/bart-large-xsum, yjernite/bart_eli5).

**Comment by tlk1997** [comment]
感谢使用，已解决问题
